### PR TITLE
multi: add loglocal flag, default to utc

### PR DESCRIPTION
**Pull request by chappjc**
This adds a `--loglocal` switch to use the local time zone in log entries, with the default to log UTC time stamps.
Without `--loglocal`, it displays the following for a reference:
`2020-09-03 19:06:50.697 [INF] MAIN: Logging with UTC time stamps. Current local time is 14:06:50 CDT`

### CODE REVIEW

**Review by JoeGruffins — APPROVED** [review]
Working well. 
My opinion: I would rather have local time the default to be honest. The flag could be --utc. This may have been discussed somewhere before.

**Comment by chappjc** [comment]
I think most people would probably choose local time.  This is me considering the practical need to be able to make any sense of other people's logs.  I figure if someone really wants to be difficult with getting support, they can use local logs.

Still, I can imagine the reverse message like

```
2020-09-03 14:06:50.697 [INF] MAIN: Logging with Local time stamps. Current UTC time is 19:06:50.697
```

But there's next to no chance I'd ever see that to figure out the delta.

Another solution might be to completely modify the time format to include timezone offset, but that's another slog change, and a bigger one.

**Comment by JoeGruffins** [comment]
I see, from a support standpoint it does make sense. This change is fine with me.

**Review by buck54321 — CHANGES_REQUESTED** [review]
Just the function signature thing, but can we also make local the default on the simnet dcrdex?

**Comment by chappjc** [comment]
Always local for simnet now:

```
$    date
Tue Sep  8 09:41:23 AM CDT 2020

$    ./dexc --simnet
2020-09-08 09:41:39.697 [DBG] CORE: new client core created

$    ./dexc --testnet
2020-09-08 14:41:52.988 [INF] DEXC: Logging with UTC time stamps. Current local time is 09:41:52 CDT

$    ./dexc --testnet --loglocal
2020-09-08 09:42:00.760 [DBG] CORE: new client core created

```

File: client/cmd/dexc/main.go [hunk 1]
```diff
@@ -56,11 +56,16 @@ func main() {
 	logStdout := func(msg []byte) {
 		os.Stdout.Write(msg)
 	}
-	logMaker := ui.InitLogging(logStdout, cfg.DebugLevel, !cfg.LocalLogs)
+	utc := !cfg.LocalLogs
+	if cfg.Net == dex.Simnet {
+		utc = false
+	}
```
Comment (buck54321) [inline client/cmd/dexc/main.go:62]: Oh. I really just meant for the dcrdex simnet harness `dex/testing/dcrdex/harness.sh`, but this is fine too.

**Review by buck54321 — APPROVED** [review]

**Pull request merged**